client/postMessageHandler.coffee

```diff
@@ -32,12 +32,16 @@ postMessageHandler = (event)->
   else if request.type == "eha.authInfo"
     Meteor.call("SetBSVEAuthTicketPassword", request, (error) ->
       if error
-        # TODO: Show Auth error banner instead
-        alert "Authentication failed: " + error
-        console.log error
+        window.parent.postMessage(JSON.stringify({
+          type: 'eha.alert',
+          msg: 'Authentication failed: EIDR-Connect',
+          dismissable: true,
+          cb: null
+        }), event.origin)
```

Comment:
We aren't logging or showing the possible error response returned from the BSVE from this
https://github.com/ecohealthalliance/eidr-connect/blob/d41fd6bae1ad6549243c7e028ba55424193ad165/server/methods.coffee#L100

Can we make sure to capture this information somewhere please